easybuild/easyconfigs/f/fastp/fastp-0.20.1-iccifort-2020.1.217.eb

```
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'fastp'
+version = '0.20.1'
+
+homepage = 'https://github.com/OpenGene/fastp'
+description = """A tool designed to provide fast all-in-one preprocessing for FastQ files.
+ This tool is developed in C++ with multithreading supported to afford high performance."""
+
+toolchain = {'name': 'iccifort', 'version': '2020.1.217'}
+
+source_urls = ['https://github.com/OpenGene/fastp/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['e1b663717850bed7cb560c0c540e9a05dd9448ec76978faaf853a6959fd5b1b3']
+
+skipsteps = ['configure']
+
+buildopts = ' CXX=${CXX}'
+
+preinstallopts = 'mkdir -p %(installdir)s/bin && '
+
+installopts = 'PREFIX=%(installdir)s'
+
+sanity_check_paths = {
+    'dirs': [],
+    'files': ['bin/fastp'],
```

Blasphemy! `files` always goes before `dirs`
```suggestion
    'files': ['bin/fastp'],
    'dirs': [],
```